Libraries/D3D12RaytracingFallback/src/FindTreelets.hlsl

```diff
@@ -0,0 +1,93 @@
+//*********************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
+// ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
+// IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
+// PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.
+//
+//*********************************************************
+
+#define HLSL
+#include "TreeletReorderBindings.h"
+#include "RayTracingHelper.hlsli"
+
+AABB ComputeLeafAABB(uint triangleIndex)
+{
+    uint2 unused;
+    Primitive primitive = InputBuffer[triangleIndex];
+    if (primitive.PrimitiveType == TRIANGLE_TYPE)
+    {
+        Triangle tri = GetTriangle(primitive);
+        return BoundingBoxToAABB(GetBoxDataFromTriangle(tri.v0, tri.v1, tri.v2, triangleIndex, unused));
+    }
+    else // if(primitiveType == PROCEDURAL_PRIMITIVE_TYPE)
+    {
+        return GetProceduralPrimitiveAABB(primitive);
+    }
+}
+
+AABB CombineAABB(AABB aabb0, AABB aabb1)
```

Comment:
I'd try to share these duplicated functions either from one of the HLSL util headers or in TreeletReorderBindings.h